webapi/src/main/scala/org/knora/webapi/responders/v1/ValuesResponderV1.scala

```diff
@@ -345,6 +349,130 @@ class ValuesResponderV1 extends ResponderV1 {
     }
 
     /**
+      * Temporary structure to represent existing file values of a resource.
+      *
+      * @param property       the property Iri (e.g., hasStillImageFileValueRepresentation)
+      * @param valueObjectIri the Iri of the value object.
+      * @param quality        the quality of the file value
+      */
+    private case class CurrentFileValue(property: IRI, valueObjectIri: IRI, quality: Option[Int])
+
+    private def changeFileValueV1(changeFileValueRequest: ChangeFileValueRequestV1): Future[ChangeFileValueResponseV1] = {
```

Comment:
What happens if two requests try to update the same file at the same time? In other update scenarios, the client has to send the IRI of the previous version of the value, and if someone else has already changed it, we reject the request. But we can't do that here, because the client don't know the IRI of the file value. This is something we need to fix in version 2 of the API. In the meantime, what does Sipi do if it receives 2 simultaneous requests to change the same file? Is there a risk that the file could be corrupted? To be on the safe side, I think this method should acquire a resource lock.